charts/ceph-csi-cephfs/templates/secret.yaml

```diff
@@ -14,6 +14,6 @@ metadata:
     heritage: {{ .Release.Service }}
     {{- with .Values.commonLabels }}{{ toYaml . | trim | nindent 4 }}{{- end }}
 stringData:
-  userID: {{ .Values.secret.userID }}
-  userKey: {{ .Values.secret.userKey }}
+  userID: {{ required "A valid userID must be specified in secret" .Values.secret.userID }}
+  userKey: {{ required "A valid userKey must be specified in secret" .Values.secret.userKey }}
```

Comment:
should we add adminID and adminKey back in this release and deprecate it and Remove it in next major release?

thoughts?